sierra_adapter/terraform/items_to_dynamo/queues.tf

```diff
@@ -6,4 +6,7 @@ module "demultiplexer_queue" {
   topic_names = ["${var.demultiplexer_topic_name}"]
 
   alarm_topic_arn = "${var.dlq_alarm_arn}"
+
+  max_receive_count = 10
```

Comment:
There should be a comment accompanying this line explaining why we’ve increased the max_receive_count (use what you wrote in the PR description).

Reply:
And it'd be useful to mark from what it was increased.

Reply:
👍